dev/ScrollBar/ScrollBar_rs4_themeresources.xaml

```diff
@@ -234,7 +238,9 @@
                                 </Grid>
                             </ControlTemplate>
                             <ControlTemplate x:Key="VerticalThumbTemplate" TargetType="Thumb">
-                                <Rectangle x:Name="ThumbVisual" Fill="{TemplateBinding Background}">
+                                <Rectangle x:Name="ThumbVisual" Fill="{TemplateBinding Background}"
```

Comment:
It looks like you've got the commits in here so I can review just the changes to these templates for CornerRadius, but it might still be better to merge to master without those first so it's easier to backtrack if anything goes wrong (since your PR will be squashed).

Reply:
> It looks like you've got the commits in here so I can review just the changes to these templates for CornerRadius, but it might still be better to merge to master without those first so it's easier to backtrack if anything goes wrong (since your PR will be squashed).

Yeah I'll create a separate PR

Reply:
I know it's a pain but it'll create a cleaner history in the long run.
Thank you!